src/components/Editor/index.tsx

```diff
@@ -271,6 +281,43 @@ const Editor = () => {
         beginLineNumToLogEventNumRef.current = beginLineNumToLogEventNum;
     }, [beginLineNumToLogEventNum]);
 
+    // On `logData`, `queryString`, `queryIsCaseSensitive`, or `queryIsRegex` update, highlight any
+    // matches.
+    useEffect(() => {
+        if (null === editorRef.current) {
+            return;
+        }
+        searchDecorationsCollectionRef.current?.clear();
+
+        const matches = editorRef.current
+            .getModel()
+            ?.findMatches(
+                queryString,
+                false,
+                queryIsRegex,
+                queryIsCaseSensitive,
+                null,
+                true
+            );
```

Comment:
sorry for missing this in the previous comment:
1. let's call this API with `captureMatches=false` so that Monaco does not spend extra effort to populate the captures groups into a `match` field. test link: http://localhost:3010/?filePath=https://yscope.s3.us-east-2.amazonaws.com/sample-logs/yarn-ubuntu-resourcemanager-ip-172-31-17-135.log.1.clp.zst#logEventNum=375558&queryString=%282015-03-24T01%3A09%3A18%29%28%5C.161%5C%2B00%3A00%29&queryIsRegex=true (there isn't any visible behaviour in the UI but if we `console.log(matches)` we can see the difference when the argument is toggled ON & OFF.
2. Let's set the optional argument `limitResultCount=Infinity`, since the default value of `999` may prevent all results on the same page from being highlighted. test link: http://localhost:3010/?filePath=https://yscope.s3.us-east-2.amazonaws.com/sample-logs/yarn-ubuntu-resourcemanager-ip-172-31-17-135.log.1.clp.zst#logEventNum=375558&queryString=2015-03-24T01%3A0

```suggestion
                false,
                Infinity
            );
```